synadm/api.py

```diff
@@ -409,6 +409,22 @@ def user_list(self, _from, _limit, _guests, _deactivated,
             "user_id": _user_id
         })
 
+    def user_list_paginate(self, _limit, _guests, _deactivated,
+                           _name, _user_id, _from="0"):
+        """Yields API responses for all of the pagination.
+
+        Args:
+            _from (string): Opaque string
+        """
+        while _from is not None:
+            response = self.user_list(_from, _limit, _guests, _deactivated,
+                                      _name, _user_id)
+            yield response
+            if "next_token" in response.keys():
+                _from = response["next_token"]
+            else:
+                _from = None
+
```

Comment:
> We try to follow the google docstrings specifications/suggestions throughout the project so far.

What document should be used as a reference for the "google docstrings"? I found [this](https://google.github.io/styleguide/pyguide.html), and I think a CONTRIBUTING.md file explaining how functions should be documented could be helpful (along with many other things you'd like to have including some things I missed in my first PR).

> Could you look up what they'd recommend for generator functions like this? Thanks!

`Yields` instead of `Returns`. https://google.github.io/styleguide/pyguide.html#s2.9-generators

Reply:
Sorry I was too lazy to google for google myself ;-) Yes, you found the right thing! And yes, a contributing.md is a missing piece in the project and I will try to work on that soonish.